stream-configuration/src/main/java/com.backbase.stream.config/BackbaseStreamConfigurationProperties.java

```diff
@@ -47,4 +48,13 @@
 
     }
 
+    @Data
+    @NoArgsConstructor
+    public static class IdentityConnectionProperties {
+        /**
+         * The location of Identity Service.
+         */
+        private String backbaseIdentityBaseUrl = "http://backbase-identity:8080";
```

Comment:
I don't like the name used not the default value, it should be called: identityIntegrationBaseUrl = "http://identity-integration-service:8080";